source/manual/upgrading-rails.html.md

```diff
@@ -0,0 +1,28 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-10-28
+title: Upgrading Rails
```

Comment:
Following the [styleguide for titles](https://docs.publishing.service.gov.uk/manual/create-page.html#title) this should probably be something like "Upgrade Rails"